### DISCUSSION TITLE
Is State list is Tenant specific or Instance specific?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#448] [Sunbird ED configuration discussion] [answered]
Can we expand this list on Sunbird Saas as we have on **"DIKSHA"**?
Because on Sunbird Saas we only 2 states.(**Refer screenshots**)
We might have pilots with teachers from other states like Maharashtra, Gujarat, Delhi, etc.
If we can expand the **state list then how we can do it**
The below image is from **DIKSHA**
![image](https://user-images.githubusercontent.com/109504832/228306241-a46ac9f9-a8c3-4042-9f6c-5e1d6c9a83f7.png)
AND the below image is from **"SUNBIRD SAAS"**
![image](https://user-images.githubusercontent.com/109504832/228306646-e793f004-2bf7-47ee-a177-adecf1e3b85f.png)
[@vinukumar-vs](https://github.com/vinukumar-vs)  [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Comment by swayangjit** [comment]
@sachinb80 These state info is been fetched from location search API(`/data/v1/location/search`). Right now it's configured only for 2 states for your instance.
@reshmi-nair Please help here with how we can populate more states in the location  Search API.

**Reply by sachinb80** [reply]
@swayangjit Thanks for the reply.
@reshmi-nair can you please help us with how we can populate all the states for our instance?

**Reply by reshmi-nair** [reply]
@sachinb80 Pls refer to this thread on how to upload location data
[DF#92](https://github.com/orgs/Sunbird-Lern/discussions/92)

**Reply by nidhikachhawa24** [reply]
@reshmi-nair 
can you please provide us the state and district list csv?

**Reply by Krishnaj20** [reply]
@nidhikachhawa24  - At Sunbird level, state and district location values will not be maintained. It is upto the adopter to get these details and populate them as per their needs.

@rhwarrier - Pls do add if you think Lern is maintaining this list.

**Comment by rhwarrier** [comment, marked as answer]
The Geo data is to be populated by the adopter - Diksha has curated the list and populated it at the instance level. Hence all tenants within the Diksha instance can use the list. The location hierarchy can be customised by instance and tenant (some states have districts and blocks , while others have clusters - for example)
